apps/quiz/views.py

```diff
@@ -9,15 +9,34 @@
 from apps.quiz.forms import QuizCreationForm
 from apps.quiz.quiz_factory import QuizFactory
 from apps.auth.permissions import IsAuthenticatedPermission
+from apps.quiz.models import Quiz
 
 
 bp = Blueprint('quiz', __name__)
 
 
 class QuizListView(BasePermissionCheckMethodView):
-
+    QUIZZES_PER_PAGE = 2
     template_name = "quiz/quiz_list.html"
 
+    def get_context(self) -> dict:
+        quizzes = Quiz.query.all()
```

Comment:
gettin all quizzes may not be the best idea, in case there are thousands of quizzes